sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/SdkInfo_ComputeManagementClient.cs

```diff
@@ -22,8 +22,9 @@ internal static partial class SdkInfo
                 new Tuple<string, string, string>("Compute", "AvailabilitySets", "2019-12-01"),
                 new Tuple<string, string, string>("Compute", "DedicatedHostGroups", "2019-12-01"),
                 new Tuple<string, string, string>("Compute", "DedicatedHosts", "2019-12-01"),
-                new Tuple<string, string, string>("Compute", "DiskEncryptionSets", "2019-11-01"),
-                new Tuple<string, string, string>("Compute", "Disks", "2019-11-01"),
+                new Tuple<string, string, string>("Compute", "DiskAccesses", "2020-05-01"),
```

Comment:
Besides, when generating from a new api version, please also update the api tags in `AzSdk.RP.props` -- they will be listed as tags of the powershell package for searching purpose.

Reply:
Hi I updated AzSdk.RP.props. Can you check if i did correctly. Thanks